### PR TITLE
+another controllers

### PR DESCRIPTION
**Pull request by Vosker**
OrderController
UserController
ShoppingCartController

### CODE REVIEW

**Review by voloshynamaryna11 — COMMENTED** [review]
good job

File: src/main/java/com/dev/cinema/controllers/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+    private final OrderMapper orderMapper;
+
+    @Autowired
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/controllers/OrderController.java]: sometimes you use annotation with the constructor, sometimes not
let's be consistent

File: src/main/java/com/dev/cinema/controllers/mappers/CinemaHallMapper.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.controllers.mappers;
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/controllers/mappers/CinemaHallMapper.java]: move all mappers to `service/mappers` folder

File: src/main/java/com/dev/cinema/controllers/mappers/ShoppingCartMapper.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+@Component
+public class ShoppingCartMapper {
+    private final TicketMapper mapper;
+
+    public ShoppingCartMapper(TicketMapper mapper) {
+        this.mapper = mapper;
+    }
+
+    public ShoppingCartResponseDto fromEntityToDto(ShoppingCart cart) {
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/controllers/mappers/ShoppingCartMapper.java]:
```suggestion
    public ShoppingCartResponseDto mapToDto(ShoppingCart cart) {
```
use uniform method names

File: src/main/java/com/dev/cinema/controllers/mappers/TicketMapper.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package com.dev.cinema.controllers.mappers;
+
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.dto.TicketResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TicketMapper {
+
+    public TicketResponseDto fromEntityToDto(Ticket ticket) {
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/controllers/mappers/TicketMapper.java]: same here

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/service/mappers/MovieSessionMapper.java [hunk 5]
```diff
@@ -29,7 +29,7 @@ public MovieSessionResponseDto toResponseFromSession(MovieSession movieSession)
         return movieSessionResponseDto;
     }
 
-    public MovieSession fromRequestToSession(MovieSessionRequestDto movieSessionRequestDto) {
+    public MovieSession mapToMovie(MovieSessionRequestDto movieSessionRequestDto) {
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/service/mappers/MovieSessionMapper.java:32]: mapToMovieSession*

**Review by KseniiaMakarova — APPROVED** [review]
and fix order of imports

**Pull request merged**